src/sentry/static/sentry/app/views/admin/installWizard/index.tsx

```diff
@@ -137,3 +134,60 @@ export default class InstallWizard extends AsyncView {
     );
   }
 }
+
+const Wrapper = styled('div')`
+  display: flex;
+  justify-content: center;
+`;
+
+const fixedStyle = css`
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+`;
+
+const Pattern = styled('div')`
+  &::before {
+    ${fixedStyle}
+    content: '';
+    background-image: linear-gradient(
+      to right,
+      ${p => p.theme.purple300} 0%,
+      ${p => p.theme.purple400} 100%
```

Comment:
I changed the colors so that they use theme colors, but not even sure if we want this?

Reply:
We should suck this style out into one of our styles files, since we use it on the first setup screens for on prem too.

Reply:
Hmm do you know which?

Reply:
I did a search for that color and there were no other usage -- maybe this is the screen you were thinking of?

Reply:
Oh doy, yes 100% it is. For some reason I was thinking this was the cli wizard screen